src/common/templates/includes/macros.njk

```diff
@@ -15,3 +15,29 @@
 		{% endif %}
 	{% endfor %}
 {% endmacro %}
+
+{% macro submit_buttons(analyticsPage, csrfToken, __) %}
```

Comment:
Bugs me a little that the i18n __ handle is a parameter. Probably lack of nunjucks knowledge there.